build/vsts-ci.yml

```diff
@@ -29,7 +35,67 @@ phases:
     displayName: build redist
   # Only build native assets to avoid conflicts.
   - script: ./build.sh -configuration $(BuildConfig) -projects $(Build.SourcesDirectory)/src/Native/Native.proj /p:TargetArchitecture=x64 /p:CopyPackageAssets=true
-    displayName: Build
+    displayName: Build Native Assets
+
+  - task: PublishBuildArtifacts@1
+    displayName: Publish Linux package assets
+    inputs:
+      pathToPublish: $(Build.SourcesDirectory)/artifacts/pkgassets
+      artifactName: PackageAssets
+      artifactType: container
+
+################################################################################
+- phase: Linux_arm
+################################################################################
+  variables:
+    BuildConfig: Release-netcoreapp3_1
```

Comment:
Why not `Release`?

Reply:
Release builds for netcoreapp 2.1 which still uses intel MKL for the CPU math libraries. We want the software fallbacks which are present in the netcoreapp 3.1 version.